docs/workflows/best-practices-architecture.md

```diff
@@ -0,0 +1,76 @@
+# Guidelines for designing a component architecture
+
+Software architecture has a central position when it comes to and managing the life cycle of large-scale software landscapes. A key part of a healthy software architecture is designing a healthy and future-proof component architecture. By incorporating functional and technical aspects in reusable blocks, component architectures allow for higher-level understanding of the software landscape as a whole. This has many benefits, but is in particular useful when considering system-wide concerns such as system evolution. However, with the ever-growing complexity and the move towards distributed and service-oriented software systems, this is becoming an increasingly challenging task to achieve.
+
+Tracking your architecture evolution is therefore key to ensuring your software landscapes and its systems remain future-proof. Every architecture is bound to change over time, due to a changing environment in which it operates. These guidelines are meant to help architects and development teams with their component architecture, based on best practices.
```

Comment:
"Tracking your architecture evolution " this suggests that you want to focus on all the changes; I think the intention is to  emphasize "having an up-to-date view of the actual/implemented architecture at all times" ?